abstract-factory/src/test/java/com/iluwatar/abstractfactory/AbstractFactoryTest.java

```diff
@@ -23,14 +23,13 @@
 
 package com.iluwatar.abstractfactory;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-import com.iluwatar.abstractfactory.App.FactoryMaker;
-import com.iluwatar.abstractfactory.App.FactoryMaker.KingdomType;
+import lombok.val;
```

Comment:
For the reasons discussed in https://github.com/iluwatar/java-design-patterns/issues/1503 I'm against using Lombok `val`.